jekyll/_cci2/runner-installation.adoc

```diff
@@ -0,0 +1,154 @@
+= Installing the CircleCI Runner
+:page-layout: classic-docs
+:page-liquid:
+:icons: font
+:toc: macro
+:toc-title:
+
+This page describes the steps required to install CircleCI Runner on Linux and macOS platforms.
+
+=== Supported Platforms
+
+At present, the supported platforms are:
+
+- Ubuntu 18.04 running on an x86_64 (AMD64) or ARM64 architecture
+- macOS Catalina (10.15.6+) or above, running on an x86_64 architecture.
+
+Running in a container is not supported at this time.
+
+=== Prerequisites
+
+==== Authentication
+
+In order to complete this process you'll need to create a namespace, resource-class and authentication token:
+
+. Install the circleci command-line tool.
+
+. Create a namespace for your organization's runner resources under:
+
+Each organization can only create a single namespace.
+
+- If you already use orbs, this will be same namespace as the orbs use. Use the following command: `circleci namespace create <name> <vcs-type> <org-name>` (e.g. if your GitHub URL is `https://github.com/circleci`, then use: `circleci namespace create my-namespace github circleci`.
```

Comment:
```suggestion
- If you already use orbs, this will be the same namespace as the orbs use. Use the following command: `circleci namespace create <name> <vcs-type> <org-name>` (e.g. if your GitHub URL is `https://github.com/circleci`, then use: `circleci namespace create my-namespace github circleci`.
```